jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -486,18 +486,24 @@ val ID_WRAPPERS =
                 ),
                 IdWrapper("PlantingSiteNotificationId", listOf("planting_site_notifications\\.id")),
                 IdWrapper(
-                    "PlantingSubzoneHistoryId",
-                    listOf("planting_subzone_histories\\.id", ".*\\.planting_subzone_history_id"),
+                    "PlantingSubstratumHistoryId",
```

Comment:
At the risk of making you redo a bunch of this: what do you think of renaming "Planting Zone" to "Stratum" rather than to "Planting Stratum?" The word "planting" is only in the existing names because "planting zone" is the name we use for the concept outside of the code (requirements, conversations with science team, etc.)

Reply:
I'm open to that, I'll check with product and Alex